### PR TITLE
Update public_suffix_list.dat .bw and .gh domains

### PR DESCRIPTION
**Pull request by mads1980**
* [ ] Description of Organization
* [ ] Reason for PSL Inclusion
* [ ] DNS verification via dig
* [ ] Run Syntax Checker (make test)

Description of Organization
====

Organization Website: 

Reason for PSL Inclusion
====

DNS Verification via dig
=======

make test
=========

### CODE REVIEW

File: public_suffix_list.dat [hunk 1]
```diff
@@ -1073,6 +1076,16 @@ edu.gh
 gov.gh
 org.gh
 mil.gh
+net.gh
```
Comment (dnsguru) [inline public_suffix_list.dat:1079]: Please resubmit the changes with these all in alphabetical order, and place the appropriate _PSL TXT record in the zone for each that matches the PR # as in the instructions.

**Comment by dnsguru** [comment]
@mads1980 When submitting these, the entries should be alphabetized and we need to validate the authority for the entries in DNS to verify the administrative contact is involved in these changes.

Please revise each of these subdomain into order, and establish the respective DNS entries.

**Comment by dnsguru** [comment]
@mads1980 **_nudge_**
> entries **should be alphabetized** and we need to **validate the authority for the entries in DNS**

**Comment by mads1980** [comment]
Unfortunately we have no authority over these domains. We noticed the issue in our production systems and cross-checked against the available resources. Feel free to close the issue, but perhaps someone is familiar enough with the .bw and .gh registries in order to get official confirmation.

**Pull request closed without merging**